src/graph/graph.c

```diff
@@ -379,9 +379,9 @@ void Graph_DeleteEdge(Graph *g, int src_id, int dest_id) {
 void Graph_LabelNodes(Graph *g, int start_node_id, int end_node_id, int label, NodeIterator **it) {
     assert(g &&
            start_node_id < g->node_count &&
-           start_node_id >=0 &&
+           start_node_id >= 0 &&
            start_node_id < end_node_id &&
-           end_node_id < g->node_count);
+           end_node_id <= g->node_count);
```

Comment:
Without this change, this assert fails on bulk inserts of a single node to succeed. I _think_ this change makes sense, but let me know if I'm misreading it!

Reply:
You're right, that fails the assert - in that scenario, the start and end ids are both 0.

Changing the assert so that `start_node_id <= end_node_id` resolves this.